src/ensembl/src/content/app/entity-viewer/components/entity-viewer-content/EntityViewerContent.scss

```diff
@@ -0,0 +1,38 @@
+@import 'src/styles/common';
+
+.entityViewerContent {
+  display: grid;
+  background-color: $black;
+  grid-template-columns: 1fr 5fr 2fr;
+  grid-template-rows: auto;
+  padding: 15px;
+  width: 100%;
+}
+
+// NOTE:
+// Remove the background colours and the heights once the PR is approved.
```

Comment:
Especially heights! :smiley: I started leaving comments to them

Reply:
Ah, I mentioned that in the description too. I should have made them bold to be easily noticeable :smile: